ryo-modal.el

```diff
@@ -63,7 +63,8 @@ add :norepeat t as a keyword."
   (mapcar (lambda (mode)
             (eval (intern-soft (concat "ryo-" (symbol-name mode) "-map"))))
           (seq-filter (lambda (elt) (or (derived-mode-p elt)
-                                        (and `(boundp ',elt) elt)))
+                                        (when (boundp elt)
```

Comment:
I think you can remove the `(when (boundp elt))` clause here, since `(and (boundp elt) (symbol-value elt))` should work.